packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -16,6 +16,8 @@ import { APIVersion, getAPIVersionFromNumber } from '@lwc/shared';
 import type { CompilerDiagnostic } from '@lwc/errors';
 
 export interface RollupLwcOptions {
+    /** A boolean indicating whether to compile for SSR runtime target. */
+    targetSSR?: boolean;
```

Comment:
Anything we add to the API surface should be documented in the README.md... even if it's just experimental for now.